README.md

```diff
@@ -76,7 +82,7 @@ The package should run on MacOS, Linux and Windows.
 To update `ggshield` you can add the option `-U/--upgrade` to the pip install command.
 
 ```shell
-$ pip install -U ggshield
+$ pip install --user -U ggshield
```

Comment:
If we explain how to install with `pipx` then we must also explain how to upgrade with it here.